src/index.html

```diff
@@ -11,12 +11,103 @@
       content="ie=edge"
```

Comment:
Keep your attributes correctly formatted. Since the `meta` tag has more than 2 attributes, each attribute should start on a new line with 2-space indentation related to the tag.

Reply:
but it is in a new line with 2 spaces